proto/projects/project.proto

```diff
@@ -28,19 +27,28 @@ message ProjectKey {
 }
 
 message Policy {
```

Comment:
should we add a comment to explain the "moretags"? e.g. the following before the messages that use it: 
```
// protobuf expected in YAML format: use "moretags" to simplify parsing
```

Reply:
done